Source/WebCore/html/canvas/CanvasRenderingContext2DBase.cpp

```diff
@@ -1659,24 +1686,24 @@ ExceptionOr<void> CanvasRenderingContext2DBase::drawImage(HTMLVideoElement& vide
 #if USE(CG)
```

Comment:
`    bool repaintEntireCanvas = rectContainsCanvas(dstRect);
`
Does this need normalizedDstRect?

Reply:
Oh! I'll bet it should! We probably need another test case to support it.